### PR TITLE
fix: unable to delete broken account

### PR DESCRIPTION
**Pull request by WofWca**
...imported from ArcaneChat, generated by a newer version of core.

Closes https://github.com/deltachat/deltachat-desktop/issues/4104.

I am not sure whether the rest of the discussion in #4104 was relevant to this issue, but I tested for a few minutes and didn't find a case where I can't delete the account or was unable to use the working accounts. Tried both with "Only Synchronize Selected Account" and without.

### CODE REVIEW

**Review by adbenitez — APPROVED** [review]
tested, works like a charm, was able to delete the account after importing it, even if IO was broken

**Review by nicodh — APPROVED** [review]
Tested and reviewed

File: src/renderer/ScreenController.tsx [hunk 1]
```diff
@@ -135,10 +135,16 @@ export default class ScreenController extends Component {
...
+    }
     runtime.setDesktopSetting('lastAccount', accountId)
-    log.info('system_info', await BackendRemote.rpc.getSystemInfo())
-    log.info('account_info', await BackendRemote.rpc.getInfo(accountId))
+    BackendRemote.rpc.getSystemInfo().then(info => {
+      log.info('system_info', info)
+    })
+    BackendRemote.rpc.getInfo(accountId).then(info => {
+      log.info('account_info', info)
+    })
```
Comment (Simon-Laux) [inline src/renderer/ScreenController.tsx:147]: what's the reason for this change?
make account switching faster by not waiting for the result? or did the calls fail?

**Review by Simon-Laux — APPROVED** [review]
Code LGTM

File: src/renderer/ScreenController.tsx (hunk 1 above)
Comment (Simon-Laux) [reply src/renderer/ScreenController.tsx:147]: acording to commit message I assume the first.

Comment (WofWca) [reply src/renderer/ScreenController.tsx:147]: It's both
